examples/nemo/eg3/Makefile

```diff
@@ -77,9 +77,15 @@ kernels:
 	${PSYCLONE} -l all -api "nemo" -s ./kernels_trans.py -opsy psy.f90 \
    ../code/tra_adv.F90
 
+# Example of using the more sophisticated script in ../scripts to
+# transform for OpenACC assuming the use of managed memory.
+# TODO #852 - simplify that script and merge it with the functionality of
+# the scripts in this directory.
 managed_mem:
-	${PSYCLONE} -l all -api "nemo" -s ./kernels_managed_mem_trans.py \
+	${PSYCLONE} -l all -api "nemo" -s ../scripts/kernels_trans.py \
    ../code/tra_adv.F90
+	PSYCLONE="${PSYCLONE}" ../scripts/process_nemo.py \
```

Comment:
Could you please explain what is going on here. You are creating a variable called PSYCLONE that contains the command to process traldf_iso.F90?

Reply:
That took me a while to work out! I've added a comment and updated the README as well as I'd forgotten to document a feature of the script.